Frends.GooglePubSub.Publish/README.md

```diff
@@ -0,0 +1,31 @@
+# Frends.GooglePubSub.Publish
+
+[![Frends.GooglePubSub.Publish Main](https://github.com/FrendsPlatform/Frends.GooglePubSub/actions/workflows/Publish_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.GooglePubSub/actions/workflows/Publish_build_and_test_on_main.yml)
+![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.GooglePubSub.Publish?label=NuGet)
+ ![GitHub](https://img.shields.io/github/license/FrendsPlatform/Frends.GooglePubSub?label=License)
+ ![Coverage](https://app-github-custom-badges.azurewebsites.net/Badge?key=FrendsPlatform/Frends.GooglePubSub|Frends.GooglePubSub.Publish|main)
+
+Returns a boolean depicting if the input text matches with the specified regular expression.
+
+## Installing
+
+You can install the Task via frends UI Task View or you can find the NuGet package from the following NuGet feed
+https://www.myget.org/F/frends-tasks/api/v2.
+
+## Building
+
+Clone a copy of the repository
+
+`git clone https://github.com/FrendsPlatform/Frends.GooglePubSub.git`
+
+Rebuild the project
+
+`dotnet build`
+
+Run tests
+
+`dotnet test`
```

Comment:
Please add instructions how to start docker container, since it is necessary for running tests.